### PR TITLE
Dummy endpoint can be deployed using AWS CDK. We are currently using Java 17 and Spring Boot 3

### PR DESCRIPTION
**Pull request by aleksandarb99**

### CODE REVIEW

File: api/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+server.port=8080
```
Comment (imflamboyant) [inline api/src/main/resources/application.properties:1]: Is this needed?

Comment (aleksandarb99) [reply api/src/main/resources/application.properties:1]: It is not. I will delete it.

File: cdk/src/test/java/com/levi9/CdkTest.java
Comment (RadosL9) [inline cdk/src/test/java/com/levi9/CdkTest.java:1]: Is this file needed?

File: commons/src/main/java/com/levi9/celebrate9/model/DummyObject.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.levi9.celebrate9.model;
+
+import java.beans.ConstructorProperties;
+
+public class DummyObject {
+    private String name;
+
+    @ConstructorProperties({"name"})
```
Comment (RadosL9) [inline commons/src/main/java/com/levi9/celebrate9/model/DummyObject.java:8]: Why is this needed?

File: functions/src/main/java/com/levi9/celebrate9/Main.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.levi9.celebrate9;
+
+public class Main {
```
Comment (RadosL9) [inline functions/src/main/java/com/levi9/celebrate9/Main.java:3]: Why do we need Main class?

File: api/src/main/java/com/levi9/celebrate9/Celebrate9Application.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import com.levi9.celebrate9.controller.DummyController;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+@Configuration
+@EnableAutoConfiguration
+@Import({ DummyController.class })
```
[the remark this answers is not in the document]
Comment (aleksandarb99) [reply api/src/main/java/com/levi9/celebrate9/Celebrate9Application.java:13]: https://github.com/awslabs/aws-serverless-java-container/wiki/Quick-start---Spring-Boot2#optimizing-cold-start-times 
Now we use the @Import annotation for importing beans and configurations. It is not recommended to use component scan when using AWS Lambda. Marko wanted us to try some optimizations, and that is the reason for this.

File: functions/src/main/java/com/levi9/celebrate9/Main.java (hunk 3 above)
Comment (aleksandarb99) [reply functions/src/main/java/com/levi9/celebrate9/Main.java:3]: We don't. I simply left the functions module blank.

**Comment by imflamboyant** [comment]
Can we have SnapStart configuration as well for this API Lambda? I think it should be here.

**Review by RadosL9 — APPROVED** [review]

**Review by imflamboyant — APPROVED** [review]
